products/focus/testing_focus.md

```diff
@@ -0,0 +1,38 @@
+# Testing on Focus for iOS and Focus for Android for a new release
+
+Given Focus for iOS and Focus for Android have very similar test procedures, this document will cover both at once.
+
+Localizers currently mostly rely on screenshots for testing. Latest information and updates about testing are provided on the [dev.l10n mailing list](https://lists.mozilla.org/listinfo/dev-l10n), which you should follow if you are working on l10n for any of the existing Mozilla products.
+
+## Testing with screenshots
+
+Screenshots are provided by the mobile teams and sent out on dev.l10n. The latest iterations can be found [right here for Focus for Android](https://npark-mozilla.github.io/index.html), [right here for Focus for iOS (iPhone5)](https://drive.google.com/drive/folders/0BwM0URyvbEeSQmZPVGdiWVlfNm8) and [right here for Focus for iOS (iPhone7)](https://drive.google.com/drive/folders/0BwM0URyvbEeSQmZPVGdiWVlfNm8).
+
+## Test builds
+
+While there are testing builds available via BuddyBuilds, the process is currently manual and localizers should reach out to the l10n mobile project manager in order to be added to these builds (currently **delphine** at **mozilla** dot **com**).
```

Comment:
A line break doesn't correspond to a line break (check the preview). You can: 
* Remove it.
* Add an empty line if you want a new paragraph.
* Add `<br/>` to create a line break.